targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/CMakeLists.txt

```diff
@@ -52,3 +54,10 @@ target_link_libraries(mbed-arm-musca-b1
         ${CMAKE_CURRENT_SOURCE_DIR}/s_veneers.o
         mbed-arm-ssg
 )
+
+mbed_post_build_tfm_sign_image(
+    ARM_MUSCA_B1
+    musca_b1
+    ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
Can you check that Greentea tests for the ARM MUSCA B1 build fine?

Reply:
I'll do that

Reply:
This line works fine. But trying to build a Greentea test is the right idea, it caught a different problem.